modules/standard/Sort.chpl

```diff
@@ -765,7 +848,8 @@ The choice of sorting algorithm used is made by the implementation.
 .. note::
 
   This function currently either uses a parallel radix sort or a parallel
-  improved quick sort.  The algorithms used will change over time.
+  improved quick sort.  For stable sort, use :proc:`sort` with ``stable=true``.
+  The algorithms used will change over time.
```

Comment:
This is the deprecated overload, I don't think it has a stable sort implemented

Reply:
Yes, the link (`:proc:'sort'`)takes you to the non-deprecated (new) overload which does have stable sort implemented.